gradle/spotbugs.gradle

```diff
@@ -10,23 +10,4 @@ allprojects {
     effort = "max"
     excludeFilter = file("$rootDir/gradle/spotbugs-exclude.xml")
   }
-
-  spotbugsTest {
```

Comment:
Oops - I still need to debug some issues I was having making this lazy configuration